swift_browser_ui_frontend/src/components/UploadModal.vue

```diff
@@ -115,7 +115,7 @@
                   rows="3"
                 />
                 <c-button
-                  type="is-success"
+                  type="success"
```

Comment:
I think there is no type of `success` for `c-button`, only type `button` or `submit`

Reply:
yes, there's no way to have other colors in the buttons. Is it ok to use default button here?

Reply:
Yes, I think the default - primary color is good enough